homeassistant/helpers/llm.py

```diff
@@ -745,7 +755,7 @@ def __init__(
         """Init the class."""
         self._domain = domain
         self._action = action
-        self.name = f"{domain}.{action}"
+        self.name = f"{domain}_{action}"
```

Comment:
why?

Reply:
I found out that some models are not happy with dots in the tool name

Reply:
can we move this to its own PR